setup/upgrade_major.rst

```diff
@@ -215,13 +213,13 @@ included in the Symfony repository for any BC break that you need to be aware of
 Upgrading to Symfony 6: Add Native Return Types
 -----------------------------------------------
 
-Symfony 6 will come with native PHP return types to (almost all) methods.
+Symfony 6 and Symfony 6 have come with native PHP return types to (almost all) methods.
```

Comment:
I changed this to "Symfony 6 and Symfony 7" while merging.

Reply:
typo my bad ^^